Dockerfile

```diff
@@ -85,7 +85,6 @@ SHELL ["/bin/bash", "-c"]
 COPY --from=builder /etc/bashrc.d /etc/bashrc.d
 COPY --from=builder /etc/bash.bashrc /etc/bash.bashrc
 COPY --from=builder /opt/scripts /opt/scripts
-COPY .bashrc /root/
```

Comment:
I had to remove this line, because `.bashrc` can not be found.

Reply:
I have moved all files that only affect the Docker image to the `.docker` subfolder.

`.docker/.bashrc` should work.

Reply:
`.docker/root/.bashrc` works